auctions/tests/model_tests.py

```diff
@@ -258,6 +259,21 @@ def test_dont_send_email_on_saving_claim(self, mock_send_mail):
 
 
 class VoteTest(TestCase):
+    def setUp(self):
+        self.user1 = mommy.make(settings.AUTH_USER_MODEL,
+                                email='user1@test.com')
+        self.user2 = mommy.make(settings.AUTH_USER_MODEL,
+                                email='user2@test.com')
+        self.user3 = mommy.make(settings.AUTH_USER_MODEL,
+                                email='user3@test.com')
+        url = 'http://test.com/bug/123'
+        issue = mommy.make(Issue, url=url)
+
+        mommy.make(Bid, user=self.user1, url=url, issue=issue, ask=50)
+        mommy.make(Bid, user=self.user2, url=url, issue=issue, offer=50)
+        mommy.make(Bid, user=self.user3, url=url, issue=issue, offer=50)
+        self.claim = mommy.make(Claim, issue=issue, user=self.user1)
```

Comment:
Is all this `setUp` close enough to the `MarketTestCase.setUp` that we could just make `VoteTest` extend from `MarketTestCase` to reduce duplication?